make-release.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+# Based on https://github.com/che-incubator/chectl/blob/master/make-release.sh
+
+set -e
+set -u
+
+usage ()
+{   echo "Usage: ./make-release.sh <schema-version> <k8s-api-version>"
+    exit
+}
+
+if [[ $# -lt 2 ]]; then usage; fi
+
+VERSION=$1
```

Comment:
Should we name `VERSION` to `SCHEMA_VERSION` and `API_VERSION` to `K8S_VERSION` to avoid confusion when reading the script?

Reply:
Yeah, that's a good point Elson, I've updated the script to be more clear in that regard.